src/state/apis/nft/types.ts

```diff
@@ -28,6 +28,7 @@ export type NftItem = {
   // The spot price of the NFT in crypto, denominated in the native asset of the NFT chain
   price: string | null
   chainId: ChainId
+  description: string | null
```

Comment:
A NFT can have its own description, different from the collection, see EDNS screenshots